jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -110,6 +110,11 @@ properties:
             type: string
             fits_keyword: ASNTABLE
             blend_table: True
+          exptype:
+            title: Exposure type provided for this exposure in the association
+            type: string
+            fits_keyword: ASNEXPTP
```

Comment:
If this is really intended only for internal use within the code and only ever accessed via the datamodel attribute, then it doesn't need to have a corresponding FITS keyword defined for it. Just leave out this one line in the schema definition. The datamodel attribute will still be there and useable in the code, but it just won't get stored in a keyword when/if the datamodel is serialized to a FITS file, which in this case is fine (IMHO). Oh, and that means the following line (blend_table) can be removed too.

Reply:
Sounds good - wasn't sure if there was a requirement to transfer these datamodel attributes to the FITS or not, but agreed that this one can stay internal.